shaders/clouds.omwfx

```diff
@@ -696,7 +719,13 @@ fragment main {
         }
 
         if (dithering_enabled) {
-            col.rgb += rand(omw_TexCoord * 1000.0) * 2.0 / 255.0;
+            vec2 ditherCoord = (omw_TexCoord.xy * omw.resolution) / 256.0; // divide by texture resolution
+            if (temporal_dithering)
+                ditherCoord += 1337.0*fract(omw.simulationTime);
+
+            float bn = omw_Texture2D(blueNoise, ditherCoord).r; // not masked by the fog, so dithering applies to the full screen. Masking it would require a lot of refactoring, or running fog functions twice.
+            float dithering = bn / dithering_strength;
```

Comment:
Could you change this to `bn * (1.0 / dithering_strength);`?

Multiplication is generally faster than division.